### DISCUSSION TITLE
No access to education benefits

### DISCUSSION THREAD

**Discussion by VladRd20** [community/community#67474] [GitHub Education]
### Hi everyone,

I've applied for the github education "pack", but i haven't got any of those benefits yet, is there any way to sort this out ?

**Comment by eminemahjoub** [comment]
u will be add ur student email and go this link https://education.github.com/pack

**Reply by VladRd20** [reply]
I have sent an application, got approved, but when I try to login to VSCode to use copilot, i get the error:
Invalid copilot token: missing token: 403

**Comment by samus-aran** [comment]
Hi @VladRd20 

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive usually within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer. Others on the community are seeing 3-4 days for receiving their academic benefits.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

We'll be closing this discussion as there are multiple discussions of a similar nature. Please use the search feature to connect with other students with related queries.
